relayer/chains/icon/listener.go

```diff
@@ -88,11 +89,12 @@ func (p *Provider) Listener(ctx context.Context, lastProcessedTx providerTypes.L
 						}
 						for _, msg := range msgs {
 							p.log.Info("Detected eventlog",
-								zap.Uint64("height", msg.MessageHeight),
-								zap.String("target_network", msg.Dst),
 								zap.Uint64("sn", msg.Sn.Uint64()),
-								zap.String("tx_hash", v.Hash.String()),
+								zap.Uint64("req_id", msg.ReqID.Uint64()),
+								zap.String("src", msg.Src),
+								zap.String("dst", msg.Dst),
 								zap.String("event_type", msg.EventType),
+								zap.Any("data", hex.EncodeToString(msg.Data)),
```

Comment:
Since the data can be huge may be we can remove it from the logging. It can be retrieved from the chain as well if required.

Reply:
Done